modules/acquia_cms_common/composer.json

```diff
@@ -81,6 +81,9 @@
             },
             "drupal/workbench_email": {
                 "3408131 - Route 'entity.workbench_email_template.canonical' does not exist.": "https://git.drupalcode.org/project/workbench_email/-/merge_requests/17.patch"
+            },
+            "drupal/acquia_connector": {
```

Comment:
@rajeshreeputra Why we added this patch ? What this is doing ?

Reply:
@vishalkhode1 this patch has been added to remove telemetry from acquia_connector module and i feel we can move this to our root composer.json. what do you say @vishalkhode1 ?

Reply:
done